README.md

```diff
@@ -57,22 +75,39 @@ Asia/Kolkata                   2024-10-06 19:20:26
 UTC                            Sunday, October 06, 2024 01:50 PM UTC
 PDT                            Sunday, October 06, 2024 06:50 AM PDT
 Asia/Kolkata                   Sunday, October 06, 2024 07:20 PM IST
-
+```
 
 4. Listing all timezones
-
-`worldtimebuddy --list`
+```
+> worldtimebuddy --list
 
 Africa/Abidjan
 Africa/Accra
 Africa/Addis_Ababa
 Africa/Algiers
 ...
-
+```
 
 5. Getting a particular timezone
-
-`worldtimebuddy --timezone UTC`
+```
+> worldtimebuddy --timezone UTC
 
 UTC                            2024-10-06 13:52:37
+...
+```
+
+6. Getting a difference of time from a particular timezone <br> (supported time formats: +/- min, hr, day) 
+`without --delta argument`
+```
+> worldtimebuddy --timezone UTC
```

Comment:
Is this example required? Can we just show an extra example in just point 5? Like

```
> worldtimebuddy --timezone JST
```

??

Reply:
done, please review